src/theme/components/template.ts

```diff
@@ -1,103 +1,75 @@
 import { defineStyleConfig } from "@chakra-ui/react";
 import { defineStyle } from "@chakra-ui/system";
-/**
- * Grid layout based on https://www.joshwcomeau.com/css/full-bleed/
- */
+
+export const responsiveGap = { base: "1rem", md: "1.5rem", xl: "1rem" };
+export const responsivePadding = { base: "1rem", md: "1.5rem", xl: "2rem" };
 
 const Template = defineStyleConfig({
   baseStyle: defineStyle({
     boxSizing: "border-box",
     color: "ui.typography.body",
-    display: "grid",
-    gridTemplateColumns: `
-      1fr
-      min(1280px, 100%)
-      1fr`,
-    rowGap: "grid.l",
     _dark: {
       color: "dark.ui.typography.body",
     },
   }),
   sizes: {},
   defaultProps: {},
 });
-// Elements that need to breakout will span outside
-// the center 1280px grid column.
-const TemplateBreakout = defineStyleConfig({
-  baseStyle: defineStyle({
-    width: "100%",
-    // This could be "1 / 4" and it would mean the same. This is
-    // "future-proof" the grid column assignment to the last column.
-    gridColumn: "1 / -1",
-  }),
-});
+
 const TemplateContent = defineStyleConfig({
   baseStyle: defineStyle({
-    // Set this element to start on the second 1280px grid column.
-    gridColumn: "2",
-    // This element also contains its own grid system within, but we use "flex"
-    // for mobile to deal with overflow issues related to the Table component.
-    display: { base: "flex", md: "grid" },
-    flexDirection: { base: "column", md: null },
-    gridTemplateColumns: "1fr",
-    paddingY: 0,
-    paddingX: "s",
-    gap: "grid.l",
+    display: "grid",
+    gridTemplateColumns: {
+      base: "repeat(1, minmax(100px, 1fr))",
+    },
+    gap: responsiveGap,
+    padding: responsivePadding,
```

Comment:
Also add the following:
```
maxWidth: "1280px"
```

This will also need something to center this element on the page. Maybe that means adding the following to `Template`?
```
display: "flex",
justifyContent: "center",
```

Or whatever works best for the overall structure of the page.